### PR TITLE
an endpoint to mark domain as porn

### PR DESCRIPTION
**Pull request by yingjieMiao**
@stkem 

2 modes: preview mode & read-write mode.

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/model/NormalizedURIRepo.scala [hunk 1]
```diff
@@ -204,4 +205,8 @@ class NormalizedURIRepoImpl @Inject() (
   def getFromId(fromId: Id[NormalizedURI])(implicit session: RSession): Seq[NormalizedURI] = {
     (for (t <- rows if t.id > fromId) yield t).list
   }
+
+  def getByRegex(regex: String)(implicit session: RSession): Seq[NormalizedURI] = {
+    (for (u <- rows if (u.url like regex)) yield u).list
```
Comment (stkem) [inline kifi-backend/modules/shoebox/app/com/keepit/model/NormalizedURIRepo.scala:210]: this does not actually do regex matching, you need to use `RLIKE`.

Comment (stkem) [inline kifi-backend/modules/shoebox/app/com/keepit/model/NormalizedURIRepo.scala:210]: not sure if that is intended or not

**Comment by stkem** [comment]
made one comment, lgtm otherwise

File: kifi-backend/modules/shoebox/app/com/keepit/model/NormalizedURIRepo.scala (hunk 1 above)
Comment (yingjieMiao) [inline kifi-backend/modules/shoebox/app/com/keepit/model/NormalizedURIRepo.scala:210]: `urlRepo` has a similar query. In fact, on the admin page, this is a separate input box. User can specify things like "porn.com". This should do it.

Comment (yingjieMiao) [inline kifi-backend/modules/shoebox/app/com/keepit/model/NormalizedURIRepo.scala:210]: I do this before the db query:

```
"%" + regex.trim + "%"
```

**Pull request merged**